dbt_project.yml

```
@@ -1,7 +1,7 @@
 config-version: 2
 
 name: 'asana_source'
-version: '0.5.1'
+version: '0.5.2'
```

Removal of fields have the possibility of breaking users downstream analytics and dashboards. Even though these fields are presumably `null`, if they are being reference downstream they will cause a failure on the users side.

As such, we will want this to actually be a breaking change.
```suggestion
version: '0.6.0'
```